### PR TITLE
Small updates on staff and staff contribution guide

### PR DESCRIPTION
**Pull request by felixfaassen**
- Added Odoo as used tool in the staff contribution guide
- Added 2 useful Git learning resource to the tools section in the staff contribution guide
- Added myself in the list of Codebase Stewards
- Fixed small type coworkers -> co-workers

-----
[View rendered contributor-guides/for-staff.md](https://github.com/publiccodenet/about/blob/contributor-guide-for-staff/contributor-guides/for-staff.md)
[View rendered organization/staff.md](https://github.com/publiccodenet/about/blob/contributor-guide-for-staff/organization/staff.md)

### CODE REVIEW

File: contributor-guides/for-staff.md [hunk 1]
```diff
@@ -14,20 +14,23 @@ Here are some important things to figure out in the first couple of days:
 * Team Public Code chat - say hello, we're a welcoming bunch
 * Calendar - add your holidays to the team calendar
 * Manual of me - fill out your own and read the existing ones
-* Github - push a branch, make a pull request (maybe to improve this page!)
+* Oodo - We use ODOO for our internal chat tool as well as project management and customer relations management
```
Comment (bvhme) [inline contributor-guides/for-staff.md:17]:
```suggestion
* Oodo - We use Odoo for our internal chat tool as well as project management and customer relations management
```

File: contributor-guides/for-staff.md [hunk 2]
```diff
@@ -14,20 +14,23 @@ Here are some important things to figure out in the first couple of days:
...
+* Github - push a branch, make a pull request (maybe to improve this page!). You can request an invite to have our Github account added to the organizational account.
 * Splitwise - if you want to join the shared lunch groceries
 * Attend morning standup ritual that is in the team calendar
 * Attend sprint retrospective & planning - review the kanbans beforehand
 * Tentoo NMBRS ESS (employee self service) mobile app - to [request leave](../activities/staff-management/leave.md), [file expenses](../activities/staff-management/expense.md) and see your payslips
 
-Any of your new coworkers will be glad to help! <3
+Any of your new co-workers will be glad to help! <3
```
Comment (bvhme) [inline contributor-guides/for-staff.md:24]: What do you think @ElenaFdR?

**Comment by felixfaassen** [comment]
thanks @bvhme ;-) smaller commits+pull requests got it!

File: contributor-guides/for-staff.md (hunk 2 above)
Comment (ElenaFdR) [reply contributor-guides/for-staff.md:24]: This is a distinction of American vs British English. We use American English at the Foundation for Public Code.

File: contributor-guides/for-staff.md [hunk 3]
```diff
@@ -14,20 +14,23 @@ Here are some important things to figure out in the first couple of days:
...
-* Github - push a branch, make a pull request (maybe to improve this page!)
+* Oodo - We use Odoo for our internal chat tool as well as project management and customer relations management
+* Github - push a branch, make a pull request (maybe to improve this page!). You can request an invite to have our Github account added to the organizational account.
 * Splitwise - if you want to join the shared lunch groceries
 * Attend morning standup ritual that is in the team calendar
 * Attend sprint retrospective & planning - review the kanbans beforehand
 * Tentoo NMBRS ESS (employee self service) mobile app - to [request leave](../activities/staff-management/leave.md), [file expenses](../activities/staff-management/expense.md) and see your payslips
 
-Any of your new coworkers will be glad to help! <3
+Any of your new co-workers will be glad to help! <3
```
Comment (ElenaFdR) [inline contributor-guides/for-staff.md:24]:
```suggestion
Any of your new coworkers will be glad to help! <3
```

**Pull request merged**